eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml

```diff
@@ -345,6 +345,10 @@
   <suppress checks="IllegalImport"
             files="com.azure.cosmos.ClientSideRequestStatistics"/> <!-- Need OperatingSystemMXBean from sun to obtain cpu info -->
 
+  <!-- management sdk suppressions -->
+  <suppress checks="[a-zA-Z0-9]*" files="[/\\]mgmt[/\\]src[/\\]test[/\\]"/>
+  <suppress checks="com.azure.tools.checkstyle.checks.EnforceFinalFieldsCheck" files="com.azure.management.resources.fluentcore.arm.ResourceId"/>
```

Comment:
At least you need to give some reason why suppress these?

If we plan to fix later, still keep it in local.

Reply:
The `EnforceFinalFieldsCheck` has already showed, "You should consider making the field final, or suppressing the warning". This is because the parameters in ResourceId are not simply set. When uses final, it will lead to compile error like `variable subscriptionId might be assigned in loop`. I'm not sure whether we need give reason, since so much suppression here without reason and it only affects our own code.

Reply:
After look at it. Maybe, I'll fix it

Reply:
Done